examples/sveltekit/README.md

```diff
@@ -0,0 +1,31 @@
+# Grafbase ⨯ SvelteKit
+
+[Join our Discord](https://discord.gg/grafbase)
```

Comment:
```suggestion
[Join our Community](https://grafbase.com/community)
```

Reply:
Are we sure about this?

Reply:
Do you think direct link is better? 
Also, we've done it for Nuxt example: https://github.com/grafbase/grafbase/pull/14#discussion_r927478978

Reply:
Okay, I'm gonna change the next one too